common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -43,6 +43,7 @@ enum eStructsConsts {
     HOSTAP_ERR_MSG_LENGTH         = 64,
     WIFI_DRIVER_VER_LENGTH        = 32 + 4,
     WIFI_SSID_MAX_LENGTH          = 32 + 1 + 3, //need extra 1 byte for null termination + alignment
+    ASSOCIATION_MAX_LENGTH        = 2048,
```

Comment:
I really would like some explanation (in the commit message or as a comment) about how you get this number.

Reply:
I was sure I explained this in the commit which changed this originally.
I don't remember the maximum size - MPDU length, I think its ~2300 Bytes.
According to the FW guys, the association frame is normally a couple of hundreds of bytes so 2KB should be well enough.

Reply:
I thought it would be nicer to add a constant variable that is set to the desired size. I can move your explained to the place where ASSOCIATION_MAX_LENGTH  is defined (beerocks_defines.h). @tomereli :)